### PR TITLE
feat: makes calendar show in user timezone

### PR DESCRIPTION
**Pull request by andrea-kyurchiev**

### CODE REVIEW

File: content/en/calendar/_index.md [hunk 1]
```diff
@@ -8,14 +8,15 @@ draft: false
...
-      </div>
+        <p>Welcome to the InnerSource Commons Calendar! Our events and working groups are open to everyone because we know that magic happens when people come together to share their experience and knowledge. </p>
+        <div class="col-md-6 order-1 order-md-2 mb-4 mb-md-0" id="calendar-container">
+        </div>
       </div>
     </div>
   </div>
 </section>
 
+<script type="text/javascript">
```
Comment (rrrutledge) [inline content/en/calendar/_index.md]: Why is this needed?

File: assets/js/script.js [hunk 2]
```diff
@@ -4,6 +4,11 @@
   // Preloader js    
   $(window).on('load', function () {
     $('.preloader').fadeOut(100);
+    if(document.getElementById('calendar-container')){
+      const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone 
+      const html = `<iframe src="https://calendar.google.com/calendar/embed?src=c_62694f414055ac569e5cb12dafbb0890ca22f3640b177a4b10b53171fbc9bdd4%40group.calendar.google.com&ctz=${timezone}" style=" border:0 " width="800" height="600" frameborder="0" scrolling="no"></iframe>`
+      document.getElementById('calendar-container').innerHTML = html;
```
Comment (rrrutledge) [inline assets/js/script.js]: A little more efficient to save the reference to the element instead of getting it again, but not a huge deal.

File: content/en/calendar/_index.md [hunk 3]
```diff
@@ -8,14 +8,15 @@ draft: false
 <section class="section">
   <div class="container">
     <div class="row align-items-center">
-        <p>Welcome to the InnerSource Commons Calendar! Our events and working groups are open to everyone because we know that magic happens when people come together to share their experience and knowledge. 
-        <p>A note about this calendar ... If you're using google, our calendar may display in the GMT timezone. To make sure you see events in your own timezone, please open the calendar on an incognito browser window or use a different search engine.
```
Comment (rrrutledge) [inline content/en/calendar/_index.md:12]: Does the calendar exist in other languages and those pages need the note removed as well?

**Review by rrrutledge — APPROVED** [review]
Thank you!  Some comments, but nothing to block merge.

File: content/en/calendar/_index.md (hunk 3 above)
Comment (andrea-kyurchiev) [reply content/en/calendar/_index.md:12]: It doesn't exist in the other languages

File: content/en/calendar/_index.md (hunk 1 above)
Comment (andrea-kyurchiev) [reply content/en/calendar/_index.md]: It isn't it was left over from testing. Removed

**Comment by rrrutledge** [comment]
This will fix https://github.com/InnerSourceCommons/InnerSourceMarketing/issues/301 as soon as it's in.

**Review by CiaraFlanagan — APPROVED** [review]
